Exygy/courtyard
# TCI-709 :: Add Page Mocks

# Pull Request

Improves Issue #TCI-709

## Issue Description

Add ability to compare design changes on dev. 

## Summary of Changes

Add page mocks for:

- Homepage
- Landing page
- Interior page
- Self-Help page

## Testing Instructions

- Visit Pattern Lab
- View Pages/*
- [ ] Each page faithfully reproduces the subject page (see [Jira](https://judasdg.atlassian.net/browse/TCI-709))
- NOTE: I wasn't able to get the offcanvas to render correctly in this iteration. I'll circle back and introduce in a future PR; I don't want to block the next step of 709.
